lib/capistrano/configuration.rb

```diff
@@ -27,6 +27,10 @@ def set(key, value)
       config[key] = value
     end
 
+    def append(key, value)
+      set(key, fetch(key, []) << value)
```

Comment:
Oopse my mistake :) @kirs do you have time? (sorry I spent the time I had this morning clearing and merging the PR backlog, otherwise I'd take this)